wiiu/vino/vino.ts

```diff
@@ -209,29 +296,29 @@ export default interface VinoAPI {
 	navi_setBaseVisibility: (unk: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk - Unknown
+	 * Enables a light blue background on the navigation box
+	 * @param light - Enable light blue background if true
```

Comment:
This is a bad parameter name. It makes it seem like a light turns on. Based on this description this enables a background, not a "light"